web/src/views/ProfileCreate.tsx

```diff
@@ -62,9 +64,22 @@ const ProfileCreate: React.FC = () => {
       // 4. Create Project Request
       await api.createProjectRequestByProfileId(profileId);
 
+      // 5.0 invite technicalCOntact to bcgov repo, only invite in production environment
```

Comment:
This is super nitpicky but probably should match the rest of the steps:
```suggestion
      // 5. invite technicalCOntact to bcgov repo, only invite in production environment
```